.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": ["rslfrkndmrky.rsl-vsc-focused-folder"]
```

Comment:
This extension was last updated 2.5 years ago. Shouldn't be on the recommended list.

Reply:
Is that bad on its own? VS Code APIs are pretty stable over time and this extension doesn't appear to have any dependencies that need updating.